frontend/src/upload.js

```diff
@@ -80,6 +80,7 @@ $(document).ready(function() {
   if (files.length === 0) {
     toggleHeader();
   } else {
+    // eslint-disable-next line no-const
```

Comment:
Sorry, I typoed in my original message. Should be `eslint-disable-next-line`. 😞 

http://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments